### PR TITLE
1 - bootstrap the app and define its feature set

### LINKED ISSUE #1: Define project scope
We need to create a document that defines the scope of this project:

- [x] What tools will be used
- [x] What features it will have 
- [x] How will the project be deployed
- [x] Bootstrap the frontend and backend

### PR DESCRIPTION
**Pull request by etowett**
# Changes

Closes #1

### CODE REVIEW

File: frontend/app/fonts/GeistMonoVF.woff
Comment (gilly7) [inline frontend/app/fonts/GeistMonoVF.woff:1]: Do we really need these fonts? We can define our own fonts

File: frontend/app/fonts/GeistVF.woff
Comment (gilly7) [inline frontend/app/fonts/GeistVF.woff:1]: same, check comments above

**Review by gilly7 — APPROVED** [review]
check comments

**Comment by etowett** [comment]
@gilly7 thanks for noting those, removed.

**Pull request merged**